circus-train-api/src/main/java/com/hotels/bdp/circustrain/api/listener/HousekeepingListener.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.hotels.bdp.circustrain.api.event;
```

Comment:
now that CT is open source we should not make this sort of changes without making deprecating the old class first - this is the API JAR. I would keep the old class, mark it with `@Deprecated` and add comment that this class must be replaced with the copy in the new package. We can then remove the class in a major release

Reply:
Normally I'd agree but we're already doing a major version bump from CT 10.0.0 to CT 11.0.0 when you compare master to the last release. Also, I'm not sure anyone outside of us is actually using CT yet so I wouldn't be as cautious as normal. The change does need to be clearly documented in the changelog though.